packages/clerk-js/src/core/modules/debug/transports/__tests__/telemetry.test.ts

```
@@ -0,0 +1,60 @@
+import { TelemetryTransport } from '../telemetry';
+import { TelemetryCollector } from '@clerk/shared/telemetry';
+import type { DebugLogEntry } from '../../types';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)